frontend/public/components/factory/table.tsx

```diff
@@ -230,13 +230,13 @@ const VirtualBody: React.SFC<VirtualBodyProps> = (props) => {
   );
```

Comment:
thanks @gnehapk - updates in `Table.tsx` look correct to me per our last discussion. I think `customData` should be `any` type and can be used to pass selection state back to the view if desired. The `event.preventDefault()` is necessary and matches PF React docs.